test/james.testing.clss.pas

```diff
@@ -0,0 +1,53 @@
+unit james.testing.clss;
+
+{$include james.inc}
+
+interface
+
+uses
+    james.testing,
+  {$IFDEF FPC}
+    fpcunit, james.testing.clss.lazarus
+  {$ELSE}
+    TestFramework, james.testing.clss.delphi
+  {$ENDIF}
+  ;
+
+type
+  {$IFDEF FPC}
+    TTestCase = FPCUnit.TTestCase;
+  {$ELSE}
+    TTestCase = TestFramework.TTestCase;
+  {$ENDIF}
```

Comment:
Any thoughts about how to solve this?
Because the goal is to remove IFDEF from other places...